### PR TITLE
new po_instance.sql

### PR DESCRIPTION
**Pull request by natalyapik**
holding_id and holding_hrid is added from po line to reflect changes in data linked from holdings back to po line

### CODE REVIEW

**Review by amzoss — CHANGES_REQUESTED** [review]
Only one small change - the column named `location_name` should be named `pol_location_name`.

All queries:
- [x] Pull request has a complete description
- [ ] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented with spaces (not tabs)
- [x] Table and column names are in all-lowercase
- [x] Quotation marks are used only where necessary
- [x] Query begins with user documentation in comment lines
- [x] File name is listed in `runlist.txt` after dependencies
- [x] All columns have indexes
- [x] Table is vacuumed and analyzed

**Review by amzoss — APPROVED** [review]
Thanks, looks good now!

- [x] Pull request has a complete description
- [x] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented with spaces (not tabs)
- [x] Table and column names are in all-lowercase
- [x] Quotation marks are used only where necessary
- [x] Query begins with user documentation in comment lines
- [x] File name is listed in `runlist.txt` after dependencies
- [x] All columns have indexes
- [x] Table is vacuumed and analyzed

File: sql/derived_tables/po_instance.sql [hunk 1]
```diff
@@ -1,49 +1,77 @@
...
 
+
 CREATE TABLE po_instance AS
 SELECT
     json_extract_path_text(po_purchase_orders.data, 'poNumber') AS po_number,
+    po_lines.po_line_number AS po_line_number,
     organization_organizations.code AS vendor_code,
     user_users.username AS created_by,
-    po_lines.po_line_number,
     po_purchase_orders.workflow_status AS po_wf_status,
```
Comment (eliana-flo) [inline sql/derived_tables/po_instance.sql:14]: I'm not sure if the abbreviation "wf" is clear enough. Maybe change to "po_workflow_status" or just "po_status"?

**Review by eliana-flo — APPROVED** [review]
Other than my small nitpick (see previous comment) this looks good and runs well.

All queries:
- [x] Pull request has a complete description
- [x] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented with spaces (not tabs)
- [x] Table and column names are in all-lowercase
- [x] Quotation marks are used only where necessary

**Review by amzoss — APPROVED** [review]
Approving again, since there was a change in a variable name

**Pull request merged**